trade-finance-manager-ui/server/nunjucks-configuration/filter-formatDateString.test.js

```diff
@@ -1,16 +1,23 @@
-import moment from 'moment';
+import { format } from 'date-fns';
 import formatDateString from './filter-formatDateString';
 
 describe('nunjuck filters - formatDateString', () => {
   it('should return formatted date from existing formatted date', () => {
-    const fromFormat = 'DD-MM-YYYY';
-    const mockDate = moment().format(fromFormat);
+    const fromFormat = {
+      moment: 'DD-MM-YYYY',
```

Comment:
Might as well bin this off once we get rid of moment :) Im assuming its done to help you!

Reply:
Yep - should've added a comment that this will need updating once functional change has since we're passing in date tokens